RELEASING.md

```diff
@@ -22,6 +22,11 @@ Feel free to use the github GUI to release, or run:
 ./scripts/release.sh
 ```
 
+## Release and sdkv2
+
+1. When releasing changes we also need to manualy update `sdkv2` branch 
+2. After release please make sure that we merge `master` branch changes into `sdkv2` branch.
```

Comment:
if this is an action that the person doing the release needs to do let's write in an active way
See https://www.mongodb.com/docs/meta/style-guide/writing/use-active-voice and https://www.mongodb.com/docs/meta/style-guide/writing/write-to-the-user/

Also since these are git command can we have the commands that can be copy/pasted? or even better can you also update the releasing script to do it for the user?

Reply:
Sounds good.